swagger.yaml

```diff
@@ -0,0 +1,99 @@
+swagger: "2.0"
+info:
+  description: "Authors Haven - A social platform for the creative at heart."
+  version: "1.0.0"
+  title: "Authors Haven"
+  license:
+    name: "MIT"
+    url: "https://www.apache.org/licenses/LICENSE-2.0.html"
+host: "kingsmen-ah-backend-staging.herokuapp.com"
+basePath: "/api"
+tags:
+- name: "user"
+  description: "User operations"
+schemes:
+  - "https"
+  - "http"
+paths:
+  /users:
+    post:
+      tags:
+      - "user"
+      summary: "Create user"
+      description: "Creates a new user in the application"
+      operationId: "createUser"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
```

Comment:
Can the input parameters be more like:

```yaml
      - name: "email"
        in: "query"
        description: "The email for registration"
        required: true
        type: "string"
        format: "email"
        minLength: 2
        maxLength: 255
      - name: "password"
        in: "query"
        description: "The password for registration"
        required: true
        type: "string",
        format: "password"
        minLength: 2
        maxLength: 255

...
```

This is just a sample. In reality, the swagger parameter definitions should match the validation definitions in https://github.com/andela/kingsmen-ah-backend/blob/develop/server/validations/auth.js

Reply:
@EMacco @emmygozi @despeauxz @adex001 kindly review my work. Thanks

Reply:
sorry for this, the `"in"` should be `"body"` not `query`